Readme.md

```
@@ -8,7 +8,7 @@ query [Circles](https://www.aboutcircles.com/) protocol events.
     * [Run a node](#run-a-node)
         * [Clone the repository](#1-clone-the-repository)
         * [Create a jwtsecret](#2-create-a-jwtsecret)
-        * [Set up the .env file](#3-set-up-the-env-file)
+        * [Set up the - .env file](#3-set-up-the-env-file)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 4; Actual: 8

(MD007, ul-indent)